docs/source/profiler.rst

```diff
@@ -600,6 +600,9 @@ Below is an breakdown of all the options.
     * **multiprocess** - Option to enable multiprocessing. Automatically selects the optimal number of processes to utilize based on system constraints.
 
       * is_enabled - (Boolean) Enables or disables multiprocessing
+      
+    * **sampling_ratio** - A percentage, as a decimal, ranging from 0 to 1 indicating how much input data to sample. Default value set to 0.2.
```

Comment:
Just noticed this, correct me if I'm wrong, but i don't think the value can be 0. So as oppose to from 0 to 1, `more than 0 and <= 1`. Not sure best way to type that up though. Does any other doc have a similar to mirror?

Reply:
good catch